webpack/assets/javascripts/react_app/components/Editor/components/EditorOptions.js

```diff
@@ -90,7 +97,9 @@ class EditorOptions extends React.Component {
               }}
               bsStyle="link"
             >
-              <Icon size="2x" type="pf" name="restart" />
+              <Icon size="md">
+                <RedoIcon />
+              </Icon>
```

Comment:
The action here is to undo the changes made in the editor so I think it will might be more correct to use `UndoIcon` (The button id is `id="undo-btn"`)